lib/y2_module_consoletest.pm

```diff
@@ -35,16 +35,8 @@ sub ncurses_filesystem_probing {
     assert_screen([('fs-probing-failed', $exit_needle)], 300);
     return unless (match_has_tag('fs-probing-failed'));
 
-    assert_screen 'fs-probing-details-btn';
-    send_key 'alt-d';
-
-    my $keymapping = {'probing-error' => 'ret', 'fs-probing-failed' => 'alt-i'};
-    foreach (qw(probing-error fs-probing-failed)) {
-        assert_screen $_;
-        send_key $keymapping->{$_};
-    }
-
-    assert_screen($exit_needle);
+    send_key 'alt-n';
+    assert_screen($exit_needle, 60);
```

Comment:
Previously a timeout of `60` wasn't necessary, why is it needed now?

Reply:
I wouldn't call it necessary, but since it should be for generic use, I have rather increased it to 1 minute.